references/auth-schemes/apiproxy/policies/APIKey.Verify.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!--
+ Copyright 2022 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+<VerifyAPIKey name="APIKey.Verify">
+    <APIKey ref="request.header.API-Key"/>
```

Comment:
Personally, I am not a big fan of the uppercase header syntax. 
I know most libs are case insensitive so not a big deal though.